adapters/mqtt-vertx/src/main/java/org/eclipse/hono/adapter/mqtt/VertxBasedMqttProtocolAdapter.java

```diff
@@ -48,9 +49,20 @@
     private static final int IANA_MQTT_PORT = 1883;
     private static final int IANA_SECURE_MQTT_PORT = 8883;
 
-    private MqttServer server;
-    private MqttServer insecureServer;
+    private MqttServer                server;
+    private MqttServer                insecureServer;
     private Map<MqttEndpoint, String> registrationAssertions = new HashMap<>();
+    private MqttAdapterMetrics        metrics;
```

Comment:
Oh - perfect.. I personally also do not like it. :) So I will change my settings..